2021-11-11 11:06:28,497 ERROR: Exception on /standard_questions/Noudke [GET] [in C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py:1457]
Traceback (most recent call last):
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\noudy\PycharmProjects\PLS\app\main\routes.py", line 122, in standard_questions
    return render_template("standard_questions.html", title='New Study', user=user, studies=studies,
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\templating.py", line 147, in render_template
    return _render(
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\templating.py", line 128, in _render
    rv = template.render(context)
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\environment.py", line 1304, in render
    self.environment.handle_exception()
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\standard_questions.html", line 2, in top-level template code
    {% from 'bootstrap/form.html' import render_form, render_field %}
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\base.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\base.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\base.html", line 1, in top-level template code
    {% extends 'base.html' %}
  [Previous line repeated 966 more times]
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\utils.py", line 544, in get
    return self[key]
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\utils.py", line 582, in __getitem__
    rv = self._mapping[key]
RecursionError: maximum recursion depth exceeded in comparison
2021-11-11 11:06:29,502 ERROR: Exception on /standard_questions/Noudke [GET] [in C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py:1457]
Traceback (most recent call last):
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\noudy\PycharmProjects\PLS\app\main\routes.py", line 122, in standard_questions
    return render_template("standard_questions.html", title='New Study', user=user, studies=studies,
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\templating.py", line 147, in render_template
    return _render(
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\flask\templating.py", line 128, in _render
    rv = template.render(context)
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\environment.py", line 1304, in render
    self.environment.handle_exception()
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\standard_questions.html", line 2, in top-level template code
    {% from 'bootstrap/form.html' import render_form, render_field %}
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\base.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\base.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\noudy\PycharmProjects\PLS\app\templates\base.html", line 1, in top-level template code
    {% extends 'base.html' %}
  [Previous line repeated 966 more times]
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\utils.py", line 544, in get
    return self[key]
  File "C:\Users\noudy\PycharmProjects\untitled2\PLS\lib\site-packages\jinja2\utils.py", line 582, in __getitem__
    rv = self._mapping[key]
RecursionError: maximum recursion depth exceeded in comparison
2021-11-11 11:07:10,818 INFO: Microblog startup [in C:\Users\noudy\PycharmProjects\PLS\app\__init__.py:77]
2021-11-11 11:14:23,446 INFO: Microblog startup [in C:\Users\noudy\PycharmProjects\PLS\app\__init__.py:77]
2021-11-11 11:16:15,675 INFO: Microblog startup [in C:\Users\noudy\PycharmProjects\PLS\app\__init__.py:77]
2021-11-11 11:17:15,102 INFO: Microblog startup [in C:\Users\noudy\PycharmProjects\PLS\app\__init__.py:77]
2021-11-11 11:17:55,506 INFO: Microblog startup [in C:\Users\noudy\PycharmProjects\PLS\app\__init__.py:77]
2021-11-11 11:18:21,202 INFO: Microblog startup [in C:\Users\noudy\PycharmProjects\PLS\app\__init__.py:77]
